suite-native/helpers/src/hooks/useRedirect.ts

```diff
@@ -0,0 +1,22 @@
+import { Linking } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { notificationsActions } from '@suite-common/toast-notifications';
+
+export const useRedirect = () => {
```

Comment:
please rename this to `useOpenLink` since there is nothing like redirect in mobile app